build.gradle

```diff
@@ -72,12 +72,6 @@ repositories {
         url "http://artifactory.terasology.org/artifactory/virtual-repo-live"
         allowInsecureProtocol true
     }
-    maven {
-        // for spf4j dependencies that haven't been merged upstream
-        // i.e. org.apache.avro:avro
-        name "spf4j dependencies"
-        url "https://dl.bintray.com/zolyfarkas/core"
```

Comment:
This bintray repository is shut down. We've published `org.apache.avro:avro:1.9.0.20p` as manual release to our Artifactory.